test/e2e_node/remote/node_e2e.go

```diff
@@ -60,7 +60,7 @@ func (n *NodeE2ERemote) SetupTestPackage(tardir, systemSpecName string) error {
 	}
 
 	// Copy binaries
-	requiredBins := []string{"kubelet", "e2e_node.test", "ginkgo", "mounter"}
+	requiredBins := []string{"kubelet", "e2e_node.test", "ginkgo", "mounter", "credential-provider"}
```

Comment:
Are we always installing the `credential-provider` binary?

Reply:
Yes, and I think it's okay since:
* going forward this should be the only mechanism for authenticating to private gcr.io images for testing
* there's some precendent for this like the gce mounter
* the node e2e remote runner is specifically for GCE node e2e tests anyways